src/commands/install.yml

```diff
@@ -4,8 +4,12 @@ description: |
 parameters:
   terraform_version:
     type: "string"
-    description: "Specify version of terraform to install."
-    default: "1.0.0" # should match executor
+    description: |
+      Version of Terraform to install. Only stable versions are supported. You can specify the version in three ways:
+      * latest: most recent version
+      * full version number (1.2.3): a specific version
+      * anything else (1.2, 1): the most recent stable version which matches this prefix.
+    default: latest
```

Comment:
Could we leave this as 1.0.0 for the time being, that we can include this change as a feature change in the next update coming soon? We can update this to latest in the next major version change we will issue shortly after.

Reply:
done!